sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -46,12 +45,11 @@ static int verify_rom_ext_identifier(rom_ext_manifest_t rom_ext) {
     return kErrorUnknown;
   }
 
-  RETURN_IF_ERROR(hmac_sha256_init(&hmac));
-  RETURN_IF_ERROR(
-      hmac_sha256_update(&hmac, &rom_ext_identifier, sizeof(uint32_t)));
+  hmac_sha256_init();
+  RETURN_IF_ERROR(hmac_sha256_update(&rom_ext_identifier, sizeof(uint32_t)));
```

Comment:
It would be nice to have an `HMAC_WARN_UNUSED_RESULT` macro.

Reply:
Done.